Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -54,7 +54,7 @@ public abstract class SharedInteractionSystem : EntitySystem
         private const CollisionGroup InRangeUnobstructedMask
             = CollisionGroup.Impassable | CollisionGroup.InteractImpassable;
 
-        public const float InteractionRange = 2f;
+        public const float InteractionRange = 1.5f;
```

Comment:
I dropped this because most interactions removed (0.35 mob radius + 0.5 tile radius) from them so technically interaction range is going up in most scenarios.